### ISSUE TITLE
Retrieve hex string representation of EthereumSignedTransaction?

### ISSUE THREAD

**Issue by blau2** [Boilertalk/Web3.swift#59]
Is there any way to get a hex string representation of the `EthereumSignedTransaction` object?

**Comment by Ybrin** [comment]
Yes. You have to convert it to an RLPItem first and then use the RLPEncoder.

```Swift
let signed = tx.sign(...)

let encoder = RLPEncoder()
let signedTxHexString = try? encoder.encode(signed).hexString(prefix: true)
```

You can add an extension for EthereumSignedTransaction which does this for you and returns the string. Maybe we should add this into the core module in an update...

**Comment by Ybrin** [comment]
This is actually exactly what `eth_sendRawTransaction` does before sending the transaction

**Comment by blau2** [comment]
Thanks @Ybrin for the quick response - exactly what I was looking for. That would be a handy thing to add but this works fine for now. Cheers!
